packages/account/src/configs.ts

```diff
@@ -3,4 +3,5 @@ export const FUEL_NETWORK_URL: string =
     ? process?.env?.FUEL_NETWORK_URL || 'http://127.0.0.1:4000/v1/graphql'
     : 'http://127.0.0.1:4000/v1/graphql';
 
+export const FUEL_DEVNET_NETWORK_URL: string = 'https://devnet.fuel.network/v1/graphql';
```

Comment:
Ni - same for `FUEL_TESTNET_NETWORK_URL`.

```suggestion
export const FUEL_DEVNET_NETWORK_URL = 'https://devnet.fuel.network/v1/graphql';
```